dockerfiles/hack/run-integration.py

```diff
@@ -55,8 +55,8 @@ def build_entry_point_args(command: click.Command, config: str,
                            dry_run: Optional[str], integration_name: str,
                            extra_args: Optional[str]) -> list[str]:
     args = ['--config', config]
-    if dry_run is not None:
-        args.append(dry_run)
+    if dry_run != '':
+        args.append('--dry-run')
```

Comment:
can `dry_run` also be `--no-dry-run`?

Reply:
My understanding is that `--no-dry-run` is the default behavior. In case I do not want to have a dry-run, then I dont set the `DRY_RUN` variable. I.e., I do not really see a use-case now for `--no-dry-run`. (I might be missing sth here though)

On the other hand one could argue that it is not a bad practice to allow very explicit flags - just to be sure things run as expected if the default changes one day silently. So I guess I will follow your advice here :+1:

Reply:
Fixed

Reply:
Dry-run change was moved to this PR https://github.com/app-sre/qontract-reconcile/pull/2119